Cubical/Tactics/Reflection.agda

```diff
@@ -0,0 +1,102 @@
+{-# OPTIONS --safe #-}
+module Cubical.Tactics.Reflection where
+
+{- Utilities common to different reflection solvers.
+
+  Most of these are copied/adapted from the 1Lab
```

Comment:
Just this file is from the 1lab. I can add a link to the file but let me know if there's a specific format I should follow.

Reply:
Yes, a link is good. And maybe "1Lab (AGPL 3.0)" to make it clear, they use a different license. I'm afraid I don't actually know how to do things like this properly.

Reply:
It would probably be good to follow the SPDX format for this, as it is quite standard now. You can have a look at https://spdx.dev/ids/ for how to do so!

Reply:
IANAL but looking at this closer, GPL is more restrictive than MIT and you can't license GPL code under MIT. I think cubical would have to also include the AGPL license, and be distributed under a dual license.

Reply:
Maybe adding something to the MIT-license file like: "Note that a more restrictive license might apply to  some individual files" is enough?

Reply:
This is now tracked as #1018. The solution there includes adding spdx identifiers, so please do that where necessary.

Reply:
I'm a tad too tired to think about doing a large-scale re-license this week, but I have no issue with treating the 1Lab.Reflection tree as BSD-3-Clause; the other contributors to those files are @ncfavier and @totbwf.

Reply:
I have absolutely no issues with having this file be BSD-3 either

Reply:
Same

Reply:
Thanks everyone for resolving this!
So we use "BSD-3-Clause" for this file.